### ISSUE TITLE
SELinux

### ISSUE THREAD

**Issue by demetrislambrou123** [horst3180/Vertex-theme#72]
It seems that on Fedora 22 alpha release with gnome 3.16 whenever I try to change from gnome-tweak-tool the gtk+ to your theme (official 3.16 version from deviantart) it creates problem with SELinux. I tried changing to other themes like numix, paper and even the default one (Adwaita) and there isn't any problem. However the theme works fine.

**Comment by horst3180** [comment]
Is there some kind of error message? Did you install the other themes manually or via the package manager?

**Comment by demetrislambrou123** [comment]
I installed both manually and via the package manager. None makes problem except yours. However after reading the error message I figured out what conflicts. the error message only comes up when I use firefox and I have a tab open that has flash video or anything related with flash. I am using freshflashplayer from https://github.com/i-rinat/freshplayerplugin it conflicts with gtkrc in the gkt-2.0 folder. Here is the error message http://pastebin.com/sk7fqSGh
